packages/KSelect/KSelect.spec.js

```diff
@@ -210,4 +216,35 @@ describe('KSelect', () => {
 
     expect(wrapper.html()).toMatchSnapshot()
   })
+
+  it('works in autosuggest mode', async () => {
```

Comment:
Can you write one additional test to make sure the filtering with `autosuggest` enabled still works?

Like, it emits the event (maybe) but at the least filters the list properly?